client/state/ui/editor/video-editor/actions.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Internal dependencies
+ */
+import {
+	VIDEO_EDITOR_RESET_STATE,
+	VIDEO_EDITOR_SET_POSTER_URL,
+	VIDEO_EDITOR_SHOW_ERROR,
+	VIDEO_EDITOR_SHOW_UPLOAD_PROGRESS,
+	VIDEO_EDITOR_UPDATE_POSTER,
+} from 'state/action-types';
+
+/**
+ * Returns an action object to be used for resetting the video editor state.
+ *
+ * @return {Object} Action object
+ */
+export const resetState = () => {
+	return {
+		type: VIDEO_EDITOR_RESET_STATE,
+	};
+};
+
+/**
+ * Returns an action object to indicate that a request has been made to update the video poster.
+ *
+ * @param {String} videoId  ID of the video
+ * @param {Object} params  Poster data
+ * @param {Number} [params.at_time]  Number of seconds into the video at which to get the poster
```

Comment:
since we're now introducing a new action in the Calypso/Redux idiom, why not follow our conventions and use `atTime` instead of `at_time`? The fact that we have to know, "Oh shoot, this function expects a value that looks like something the API wants…" makes me think that we're likely coupling concerns which would be best left separate.

Thanks for documented the properties though!